docs/source/features/features.rst

```diff
@@ -0,0 +1,233 @@
+.. _label_features:
+
+********
+Features
+********
+
+Appleseed supports amongst other features
+
+    * fully programmable shading via `Sony Pictures Imageworks <http://www.imageworks.com/>`_’ `Open Shading Language (OSL) <https://github.com/imageworks/OpenShadingLanguage>`_ 
+    * RGB/spectral/mixed rendering
+    * fast and robust transformation and deformation motion blur
+    * state-of-the-art ray traced subsurface scattering
+    * exhaustive Python and C++ APIs
+    * and many other production-oriented features.
+
+.. _label_features_ecosystem:
+
+Ecosystem
+=========
+
+Along with the core renderer, the team is actively developing high quality integrations
+
+    * appleseed for Autodesk® Maya® → `appleseed-maya <https://github.com/appleseedhq/appleseed-maya>`_
+    * appleseed for Autodesk® 3ds Max® → `appleseed-max <https://github.com/appleseedhq/appleseed-max>`_
+    * appleseed for Blender®  → `blenderseed <https://github.com/appleseedhq/blenderseed>`_ 
+
+appleseed is also the default rendering engine of `Image Engine’s Gaffer <http://www.gafferhq.org/>`_.
+
+.. _label_features_detailed_features_list:
+
+Detailed Features List
+======================
+
+.. _label_features_light_transport:
+
+Light Transport
+---------------
+
+ * Unidirectional Path Tracing [#]_
+ * Stochastic Progressive Photon Mapping [#]_
+ * Light Tracing
+
+.. _label_features_reflection_models:
+
+Reflection Models
+-----------------
+
+ * Lambertian BRDF (purely diffuse)
+ * Specular BRDF (perfect mirror)
+ * Specular BSDF
+ * Diffuse BTDF
+ * Oren-Nayar Microfacet BRDF
+ * Ward Microfacet BRDF
+ * Blinn Microfacet BRDF
+ * Microfacet BRDF with GGX, GTR, Student's t-MDF
+ * Disney's principled BRDF
+ * Ashikhmin-Shirley BRDF
+ * Kelemen BRDF
+ * Glossy BRDF
+ * Conductor Fresnel metal BRDF
+ * Stacked plastic BRDF
+ * Glass BSDF with absorption
+ * Sheen BRDF
+ * Arbitrary mixtures of BRDFs
+ * Energy compensation in microfacet BxDFs.
+
+.. _label_features_subsurface_scattering:
+
+Subsurface Scattering
+---------------------
+
+Appleseed provides state of the art raytraced, animation-friendly subsurface scattering, fully exposed to OSL.
```

Comment:
*appleseed* (lower case)

*state-of-the-art*

Reply:
Even in the beginning of a sentence *appleseed* remains without capitalization of the 1st letter?

Reply:
in terms of features, i think the website generalizes a bit more without going into details, and the docs here should be a bit more specific. For instance, we have no mention about other MDFs, we only refer to the GGX MDF, and yet we support GTR and Student's t-MDF, albeit energy conservation is active only for GGX, Beckmann, just to name an example.
What do you think? Have in mind the crux of this PR is providing the base initial configuration :)

Reply:
> Even in the beginning of a sentence appleseed remains without capitalization of the 1st letter?

So far that's what we've done indeed. The only exception is https://www.blender.org/download/external-renderers/ because Ton kindly insisted :)

Reply:
Ok, done. I'll review the appleseed-maya docs as well to make sure we're consistent in this regard.

Reply:
> we have no mention about other MDFs, we only refer to the GGX MDF, and yet we support GTR and Student's t-MDF

That's an omission, it should be fixed on the website's feature list.

> energy conservation is active only for GGX, Beckmann, just to name an example.

We probably should avoid having nearly identical feature lists in two distinct places as this will inevitably lead to them becoming partially out-of-date and/or out-of-sync.

However specific aspects such as compatibility between features (the example you gave) totally have their place in the docs.